src/tests/Interop/COM/ComWrappers/API/ComWrappersTests.csproj

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <NativeAotIncompatible>false</NativeAotIncompatible>
```

Comment:
Why is this needed? I would not expect us to have to enable ComWrappers tests one-by-one (it runs the risk of new ComWrappers tests not running with nativeaot by accident.)

Reply:
ComWrappers on NativeAOT only supports the basic API scenarios, so some of the other tests require features that have yet to be implemented. Also some of the tests require app manifests that I don't believe work with the existing binplace mechanism for the NativeAOT runtime tests.

Reply:
What I have trouble with is finding the spot that sets this to true for this project - why do we need to set it to false?

Reply:
It's set to true in Directory.Build.props in Interop/COM.

Reply:
That one is conditioned on IsManagedCOMClient so I would not expect it to apply

Reply:
I'll try to figure out why these tests weren't being run otherwise.

Reply:
I think they build - NativeAotIncompatible prevents things from building but here's a log from #84404 https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/237437/logs/1914 that has:

```
2023-04-13T02:33:59.7181103Z D:\a\_work\1\s\artifacts\TargetingPack\xunit.assert.dll : warning IL2104: Assembly 'xunit.assert' produced trim warnings. For more information see https://aka.ms/dotnet-illink/libraries [D:\a\_work\1\s\src\tests\Interop\COM\ComWrappers\API\ComWrappersTests.csproj] [D:\a\_work\1\s\src\tests\build.proj]
```

so we do look at it.

Reply:
Looks like issues.targets has the exclusion. I'll remove it from there.